frontend/src/cljs/imas_seamap/db.cljs

```diff
@@ -30,14 +30,25 @@
                                        :trigger :map.logic.trigger/automatic}
                      :controls        {:transect false
                                        :download nil}
-                     :boundaries      {:networks         []
-                                       :active-network   nil
-                                       :parks            []
-                                       :active-park      nil
-                                       :zones            []
-                                       :active-zone      nil
-                                       :zones-iucn       []
-                                       :active-zone-iucn nil}
+                     :boundaries      {:active-boundary nil
+                                       :amp             {:networks         []
+                                                         :parks            []
+                                                         :zones            []
+                                                         :zones-iucn       []
+                                                         :active-network   nil
+                                                         :active-park      nil
+                                                         :active-zone      nil
+                                                         :active-zone-iucn nil}
+                                       :imcra           {:provincial-bioregions       []
+                                                         :mesoscale-bioregions        []
+                                                         :active-provincial-bioregion nil
+                                                         :active-mesoscale-bioregion  nil}
+                                       :meow            {:realms           []
+                                                         :provinces        []
+                                                         :ecoregions       []
+                                                         :active-realm     nil
+                                                         :active-province  nil
+                                                         :active-ecoregion nil}}
```

Comment:
Aah, that makes a bit more sense -- the namespace "pollution" in the subs irks a little (it feels like [:msubs/boundaries <boundarytype>] might make more sense; still could work, but perhaps a bit more care needed here?)